packages/contracts/SETUP.md

```diff
@@ -0,0 +1,62 @@
+# Setup
+
+## Terminology
+
+- **Program Operators**: wallets that have the permission to create & manage the program
+- **Program**: maintained by Program Operators which together form a team
+- **Round Operators**: wallets that have the permission to create & manage the round
+- **Round**: created by a Program and deal with everything relating to running a round
+- **Voter** : wallet who cast a vote to a grant during the round
```

Comment:
^ Would this belong in the setup ? It was originally intended to be more product documentation but we never got to that

Reply:
I'm going to remove it and have product documentation live in a Gitbook